README.md

```
@@ -232,6 +232,37 @@ server {
+
+WStunnel server provides a `/_stats` endpoint that displays information about connected tunnels. When accessed from localhost, it provides detailed information including:
+
+- Number of active tunnels
+- Token information for each tunnel
+- Pending requests per tunnel
+- Client IP address and reverse DNS lookup
+- Client version information
+- Idle time for each tunnel
+
+Example output:
+```text
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)